portfolio/src/main/webapp/script.js

```diff
@@ -17,9 +17,62 @@ function init_index() {
 }
 
 function init_games() {
+    getLogin();
     getGameTabs();
 }
 
+function getLogin() {
+    var loginButton = document.getElementById('game-login-button');
+    var loginText = document.getElementById('game-login-text');
+
+    fetch('/login').then(response => response.text()).then((response) => {
+      var responseSplit = response.split("\n");
+      var message = responseSplit[0];
```

Comment:
I would advice against relying on line ordering to consume the response.  This makes it hard to add more response lines, and keep track of which is which since you'll have to keep both your client and your server side in sync.  Better approach would be to send the response back in xml or json format.